Assets/Scripts/Score/OverallScoreManager.cs

```diff
@@ -0,0 +1,21 @@
+using UnityEngine;
+using TMPro;
+
+public class OverallScoreManager : MonoBehaviour
+{
+    [SerializeField]
+    private TextMeshProUGUI highScore;
+
+    private float overallScore;
+    private int lastLevel = 2;
+
+    private void Awake()
+    {
+        for (int i = 0; i <= lastLevel; i++)
```

Comment:
I would prefer for the infinite level score to be stored with a PlayerPrefs key that is clearer than `lvl0HighScore`, but changing that does complicate the implementation. I'm fine with keeping this implementation. `lvl0HighScore` being used for the infinite level score should be documented with a comment here.

Reply:
Unsure how I would do that without making it's own script, also unsure if that would be worth the extra script.

Reply:
Realized can do this through Serializable field, mbad